### PR TITLE
Add keywords

### LINKED ISSUE #86: Add search keywords

### PR DESCRIPTION
**Pull request by leonid-sviderskii-mtl**
Closes #86

### CODE REVIEW

**Comment by leonid-sviderskii-mtl** [comment]
@vrfloppa please check again!

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 	<link href="https://emoji-css.afeld.me/emoji.css" rel="stylesheet">
 
-	<title>Engineer-Petr 🌎🚜🇷🇺🇺🇦🇧🇾</title>
+	<title>Engineer-Petr: Relocation for IT professionals 🌎🚜🇷🇺🇺🇦🇧🇾</title>
```
Comment (vrfloppa) [inline index.html:25]: 👍

**Review by vrfloppa — APPROVED** [review]
Great stuff!

**Comment by leonid-sviderskii-mtl** [comment]
@vrfloppa I guess only you can merge it, right?

**Pull request merged**